.github/workflows/cloud_regression.yml

```diff
@@ -22,21 +22,24 @@ jobs:
           PR_COMMIT_HASH: ${{ github.event.pull_request.head.sha }}
         id: output-workflow-dispatch-params
         run: |
-          if [ ${{ github.event_name }} == 'pull_request_target' ]; then
+          if [ "${github.event_name}" == 'pull_request_target' ]; then
             NETDATA_CUSTOM_REPO="$PR_REPO_NAME"
             NETDATA_CUSTOM_BRANCH="$PR_BRANCH_NAME"
-            NETDATA_CUSTOM_PR_NUMBER="${{ github.event.number }}"
+            NETDATA_CUSTOM_PR_NUMBER="${github.event.number}"
             NETDATA_CUSTOM_COMMIT_HASH="$PR_COMMIT_HASH"
-          elif [ ${{ github.event_name }} == 'push' ]; then
+          elif [ "${github.event_name}" == 'push' ]; then
             NETDATA_CUSTOM_REPO="netdata/netdata"
             NETDATA_CUSTOM_BRANCH="master"
             NETDATA_CUSTOM_PR_NUMBER=""
-            NETDATA_CUSTOM_COMMIT_HASH="${{ github.sha }}"
+            NETDATA_CUSTOM_COMMIT_HASH="${github.sha}"
           fi
```

Comment:
These changes are incorrect, we need the doubled curly braces because the substitutions are being done by the GitHub actions environment, not the shell.

Reply:
my bad, fixed in ee8f851015612e93409a00d461d27102e46e26e7